FreydCategoriesForCAP/gap/RingsAsAbCats.gi

```diff
@@ -127,6 +141,17 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_RING_AS_CATEGORY,
     
     ring := UnderlyingRing( category );
     
+    ##
+    AddMorphismConstructor( category,
```

Comment:
Please either
* remove this installation again
* also install `MorphismDatum`.

Reply:
Done by `AddMorphismDatum`.

Reply:
Please also add a test for `AddMorphismDatum` so we keep the 100% code coverage.

Reply:
Done.